[2022-06-06 04:14:50,480] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: hero_info.hero_external_table_task scheduled__2022-06-05T04:00:00+00:00 [queued]>
[2022-06-06 04:14:50,538] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: hero_info.hero_external_table_task scheduled__2022-06-05T04:00:00+00:00 [queued]>
[2022-06-06 04:14:50,538] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-06 04:14:50,539] {taskinstance.py:1358} INFO - Starting attempt 1 of 2
[2022-06-06 04:14:50,539] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-06 04:14:50,550] {taskinstance.py:1378} INFO - Executing <Task(BigQueryCreateExternalTableOperator): hero_external_table_task> on 2022-06-05 04:00:00+00:00
[2022-06-06 04:14:50,558] {standard_task_runner.py:52} INFO - Started process 9398 to run task
[2022-06-06 04:14:50,563] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'hero_info', 'hero_external_table_task', 'scheduled__2022-06-05T04:00:00+00:00', '--job-id', '15124', '--raw', '--subdir', 'DAGS_FOLDER/ingest_heroes.py', '--cfg-path', '/tmp/tmp8gq6a9a6', '--error-file', '/tmp/tmprdzv7444']
[2022-06-06 04:14:50,564] {standard_task_runner.py:80} INFO - Job 15124: Subtask hero_external_table_task
[2022-06-06 04:14:50,623] {task_command.py:370} INFO - Running <TaskInstance: hero_info.hero_external_table_task scheduled__2022-06-05T04:00:00+00:00 [running]> on host 76384e2d8475
[2022-06-06 04:14:50,777] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=hero_info
AIRFLOW_CTX_TASK_ID=hero_external_table_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-05T04:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-05T04:00:00+00:00
[2022-06-06 04:14:50,779] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-06-06 04:14:50,780] {base.py:68} INFO - Using connection ID 'google_cloud_default' for task execution.
[2022-06-06 04:14:51,256] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=hero_info, task_id=hero_external_table_task, execution_date=20220605T040000, start_date=20220606T041450, end_date=20220606T041451
[2022-06-06 04:14:51,299] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-06 04:14:51,353] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
